book/03_Geospatial_data_files/geographic_data_formats.md

```diff
@@ -60,6 +60,15 @@ A GeoTIFF file extension contains geographic metadata that describes the actual
 Geodata is drawn from vector formats on a map, and the geodata is converted to the specified output projection of the map if the projection in the source file differs. Some of the vector and raster formats typically supported by a GeoTIFF online viewer include: asc, gml, gpx, json, kml, kmz, mid, mif, osm, tif, tab, map, id, dat, gdbtable, and gdbtablx.^2^
 
 
+## A Note on the Ordering of Coordinates in Python GIS Libraries
```

Comment:
This is a good explanation; it belongs earlier in the tutorial. I'll merge this commit and cut-and-paste this into the appropriate place when I know where that is.